crates/subspace-networking/src/behavior/persistent_parameters.rs

```diff
@@ -341,6 +341,7 @@ impl KnownPeersManager {
             .read(true)
             .write(true)
             .create(true)
+            .truncate(false)
```

Comment:
was this not a default before or are we being explicit here ?

Reply:
Being explicit due to new lint: https://rust-lang.github.io/rust-clippy/master/index.html#/suspicious_open_options